insights/collect.py

```diff
@@ -241,6 +241,10 @@
       enabled: true
     - name: insights.components.selinux.SELinuxEnabled
       enabled: true
+
+    # needed for the 'ls lan' spec
```

Comment:
Better explain it a bit clearer, e.g. `needed for the 'spec.fstab_mounted' to the 'ls_lan' spec`